Makefile

```diff
@@ -58,43 +71,40 @@ EFM8_LOAD_BAUD	?= 115200
 .NOTPARALLEL:
 
 define MAKE_OBJ
-OBJS += $(1)_$(2)_$(3)_$(4)_$(VERSION).OBJ
-$(OUTPUT_DIR)/$(1)_$(2)_$(3)_$(4)_$(VERSION).OBJ : $(ASM_SRC) $(ASM_INC)
+OBJS += $(1)_$(2)_$(3)_$(VERSION).OBJ
+$(OUTPUT_DIR)/$(1)_$(2)_$(3)_$(VERSION).OBJ : $(ASM_SRC) $(ASM_INC)
 	$(eval _ESC			:= $(1))
 	$(eval _ESC_INT		:= $(shell printf "%d" "'${_ESC}"))
 	$(eval _ESCNO		:= $(shell echo $$(( $(_ESC_INT) - 65 + 1))))
 	$(eval _MCU_TYPE	:= $(subst L,0,$(subst H,1,$(subst X,2,$(2)))))
 	$(eval _DEADTIME	:= $(3))
-	$(eval _PWM_FREQ	:= $(subst 24,0,$(subst 48,1,$(subst 96,2,$(4)))))
 	$$(eval _LST		:= $$(patsubst %.OBJ,%.LST,$$@))
 	@mkdir -p $(OUTPUT_DIR)
 	@echo "AX51 : $$@"
 	@$(AX51) $(ASM_SRC) \
 		"DEFINE(ESCNO=$(_ESCNO)) " \
 		"DEFINE(MCU_TYPE=$(_MCU_TYPE)) "\
 		"DEFINE(DEADTIME=$(_DEADTIME)) "\
-		"DEFINE(PWM_FREQ=$(_PWM_FREQ)) "\
+		"DEFINE(PWM_FREQ=0) "\
```

Comment:
Does this external device make sense still? If it's constant anyway, I think we can set it in the asm file directly.

Reply:
I wanted to leave it for development current version, but it is not really necessary. I will also remove it

Reply:
done